networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="configuring-ingress-cluster-traffic-aws"]
+= Configuring ingress cluster traffic on AWS
+include::_attributes/common-attributes.adoc[]
+:context: configuring-ingress-cluster-traffic-aws
+
+toc::[]
+
+{product-title} provides methods for communicating from outside the cluster with services running in the cluster. This method uses load balancers on AWS, specifically a Network Load Balancer (NLB) or a Classic Load Balancer (CLB). Both types of load balancers can forward the client's IP address to the node, but a CLB requires additional link:https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-proxy-protocol.html[configuring of proxy protocol support].
```

Comment:
@Miciah lmk what you think of this updated text. The level of detail matches what existed before, but definitely let me know if users would expect or want more detailed information here.

Reply:
681f15a..d24453f clarify that OCP enables proxy protocol support by default